### PR TITLE
ADDED CODECHEF JUNE LONG CHALLNEGE DIV 3 SOLUTION IN C++

### PR DESCRIPTION
**Pull request by codeaholic-shub**
Hello @AkashSingh3031 ,
Kindly Review this PR !

### CODE REVIEW

File: 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/MINIMUM_DUAL_AREA.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#include<bits/stdc++.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/MINIMUM_DUAL_AREA.cpp:1]: Accepted

File: 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/MINIMUM_SUBTREE_COVER.cpp [hunk 2]
```diff
@@ -0,0 +1,96 @@
+#include <bits/stdc++.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/MINIMUM_SUBTREE_COVER.cpp:1]: Accepted

File: 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/SUMMER_HEAT.cpp [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#include<bits/stdc++.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/SUMMER_HEAT.cpp:1]: Accepted

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
Please check your solution of the 3 problems `Bella ciao`, `Bitwise Tuples`, `Shortest Route`

**Comment by AkashSingh3031** [comment]
@codeaholic-shub 
After changing the code again send a Pull Request

**Comment by codeaholic-shub** [comment]
> Please check your solution of the 3 problems `Bella ciao`, `Bitwise Tuples`, `Shortest Route`

I have already checked , its already submitted solutions of codechef from my account

File: 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/BITWISE_TUPLES.cpp [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#include<bits/stdc++.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/BITWISE_TUPLES.cpp:1]: Accepted

File: 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/BELLA_CIAO.cpp [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#include<bits/stdc++.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/BELLA_CIAO.cpp:1]: image

check again

File: 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/SHORTEST_ROUTE.cpp [hunk 6]
```diff
@@ -0,0 +1,124 @@
+
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/SHORTEST_ROUTE.cpp:1]: image

Check again

**Comment by codeaholic-shub** [comment]
Hello @AkashSingh3031 ,
I have changed the solution and its correct,
U may check my PR !

File: 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/BELLA_CIAO.cpp [hunk 7]
```diff
@@ -0,0 +1,29 @@
+#include <bits/stdc++.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/BELLA_CIAO.cpp:1]: Accepted

File: 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/SHORTEST_ROUTE.cpp [hunk 8]
```diff
@@ -0,0 +1,78 @@
+#include <bits/stdc++.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/JUNE_LONG_CHALLENGE_DIV_3/SHORTEST_ROUTE.cpp:1]: Accepted

**Review by AkashSingh3031 — COMMENTED** [review]
All Accepted

**Pull request merged**